packages/edit-site/src/components/template-details/style.scss

```diff
@@ -0,0 +1,12 @@
+.edit-site-template-details {
+	margin: 0 $grid-unit-10;
+}
+
+.edit-site-template-details__heading {
+	margin-top: $grid-unit-05;
+	margin-bottom: $grid-unit-15;
+	color: $gray-700;
+	text-transform: uppercase;
+	font-size: 11px;
```

Comment:
`Non-blocking`:

I think it's fair to have a font-size that's smaller than the default `13px` because it's uppercase. Is there something we could add to `packages/base-styles/_variables.scss` to standardize this?

Reply:
I think this whole style needs standardized (I'm copying something that's hardcoded in a few other files here), so I think it's outside of the scope of this PR. but something that could be done in a follow-up!